RecoLocalMuon/GEMSegment/plugins/ME0SegAlgoRU.cc

```diff
@@ -380,12 +380,8 @@ bool ME0SegAlgoRU::areHitsCloseInEta(const float maxETA,
   edm::LogVerbatim("ME0SegAlgoRU") << "[ME0SegAlgoRU::areHitsCloseInEta] gp1 = " << h1 << " in eta part = " << h1.eta()
                                    << " and gp2 = " << h2 << " in eta part = " << h2.eta() << " ==> dEta = " << diff
                                    << " ==> return " << (diff < 0.1) << std::endl;
-  return (
-      diff <
-      std::max(
-          maxETA,
-          float(
-              0.01)));  //temp for floating point comparision...maxEta is the difference between partitions, so x1.5 to take into account non-circle geom.
+  return ( diff < std::max( maxETA, 0.01f));
+//temp for floating point comparision...maxEta is the difference between partitions, so x1.5 to take into account non-circle geom.

```

Comment:
a comment aligned with the code and before the action is done seem to be more common.

BTW, does the text even make sense?
- ```temp for floating point comparision...``` sounds like it's justifying why ```float``` was added in the original code
- what is this "x1.5 ... " about?

Reply:
> BTW, does the text even make sense?
> 
> * `temp for floating point comparision...` sounds like it's justifying why `float` was added in the original code
> * what is this "x1.5 ... " about?

I wondered the same, but it was not my intention to inquire about it here.
As you know, even ```diff``` here makes no se sense (as it is a difference of ```h1.eta()``` with itself. But fixing that bug was not the scope of this PR

Reply:
the code was modified (```float (0.01)``` -> ```0.01f``` ) and it apparently affects the content of the comment.